plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt

```diff
@@ -57,24 +60,40 @@ fun androidLibraryFeatureDefinition(
     }
 )
 
+private fun LibraryExtension.validateManifestPackage(
+    packageName: String,
+    project: Project
+) {
+    sourceSets.forEach {
+        if (it.manifest.srcFile.exists()) {
+            val manifestPackageName = DocumentBuilderFactory
+                .newInstance()
+                .newDocumentBuilder()
+                .parse(InputSource(it.manifest.srcFile.reader()))
+                .documentElement
+                .getAttribute("package")
+
+            if (manifestPackageName != packageName) {
+                error(
+                    "Manifest package != build.gradle.kts packageName, it must be equal" +
+                            "\nmanifest ${it.manifest.srcFile.absolutePath}" +
+                            "\npackage $manifestPackageName" +
+                            "\nbuild script ${project.buildFile.absolutePath}" +
+                            "\npackageName $packageName"
+                )
+            }
+        }
+    }
+}
+
 private fun LibraryExtension.maybeGenerateManifest(
     project: Project,
     feature: AndroidLibraryFeatureConfiguration
 ) {
-    val manifestFile = manifestFile(project.buildDir, feature.packageName)
+    val manifestFile = File(project.projectDir, "src/main/AndroidManifest.xml")
```

Comment:
There could be a few cases where we'll need different manifests in `src/smth` folders do we want to support that?